### ISSUE TITLE
Implement ${!PREFIX_*} and ${!PREFIX_@}

### ISSUE THREAD

**Issue by andychu** [oilshell/oil#545]
Used by homebrew apparently

```
               "${!HOMEBREW_@}" "${!TRAVIS_@}"
                            ^
/usr/local/bin/brew:93: Expected } after var sub, got '@'
```

https://oilshell.zulipchat.com/#narrow/stream/119655-new-members/topic/Testing.20osh

**Comment by Crestwave** [comment]
I'm not aware of any `${PREFIX_@}` syntax; this is only valid in Bash because it interprets it as `${parameter@operator}`. This is `${!prefix*}` and `${!prefix@}`.

**Comment by andychu** [comment]
Released with 0.7.pre7
